lambdas/pkgpush/index.py

```diff
@@ -229,6 +281,15 @@ def from_botocore_error(cls, boto_error: ClientError):
         return cls(status_code, message)
 
 
+class FileTooLargeForHashing(ApiException):
+    def __init__(self, logical_key):
+        super().__init__(
+            HTTPStatus.BAD_REQUEST,
+            f'Package entry {logical_key!r} is too large for hashing. '
+            f'Max size is {S3_HASH_LAMBDA_MAX_FILE_SIZE} bytes.'
```

Comment:
This is a good example of why units are useful in variable names.

`S3_HASH_LAMBDA_MAX_FILE_SIZE_BYTES` would be clearer.

Reply:
> This is a good example of why units are useful in variable names.

I'm fine with including units in variable name, but don't quite get why this is a good example. :thinking:

Reply:
well, reader can't know that bytes is the right unit for the f-string without more research into units of variable